Makefile

```diff
@@ -26,6 +26,8 @@ clean-test: ## remove test and coverage artifacts
 initdb:
 	docker-compose exec db psql -h db -d postgres -U postgres -c "DROP DATABASE IF EXISTS ${DB_NAME}"
 	docker-compose exec db psql -h db -d postgres -U postgres -c "CREATE DATABASE ${DB_NAME} ENCODING 'UTF8'"
+	docker-compose exec db psql -h db -d postgres -U postgres -c "DROP DATABASE IF EXISTS tests"
+	docker-compose exec db psql -h db -d postgres -U postgres -c "CREATE DATABASE tests ENCODING 'UTF8'"
```

Comment:
Ah, that seems to be because that's the default for `make_database_url` function. https://github.com/openstax/qualitas-server/blob/master/qualitas/utils.py#L16-L27

It fails over to tests b/c i'd rather not wipe out the regular database by accident.

I should probably make the DB_NAME in instance/conf.py default to qualitas_db.

Reply:
Oh... I see!  I'll change instance/conf.py.  I was also wondering if it's ok to change `DB_HOST` to `db` by default, or perhaps better yet, put this in docker-compose.yml.

Reply:
yea, that's better. I did this on my cookiecutter-flask project but didn't move the change over here.